yjit/bindgen/src/main.rs

```diff
@@ -78,6 +78,10 @@ fn main() {
 
         .allowlist_function("rb_attr_get")
 
+        .allowlist_function("rb_yjit_mark_writable")
+        .allowlist_function("rb_yjit_mark_executable")
+        .allowlist_function("rb_yjit_get_page_size")
```

Comment:
What do you think about just whitelisting `rb_yjit_*` ?

Reply:
Well, being super pedantic here again 😅 but I'm thinking we use `rb_yjit_` for anything that we specifically define as part of YJIT, something we create for our own needs. Whereas `rb_` is the generic prefix for any global symbol in CRuby.

Reply:
Yeah exporting `rb_yjit_*` makes sense to me. A quick search reveals that rust-bindgen doesn't offer an API to add custom `#[link_name]` attributes for the purposes of renaming. If we really want to do the renaming we could build it into the tool with the `syn` Rust parser and the `prettyplease` unparser. For now though I think sticking to the simple but a bit verbose prefix is okay.

Reply:
Yeah, simple but verbose makes sense to me.

We're also planning to add a build step for YARV opcodes -- if need be, we could pretty easily make a list of declarations and generate a Rust source file from it. Not sure it's worth the time, but it wouldn't be hard.